test/unit_test_helper.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_helper'
+require 'webmock'
+
+include WebMock::API
+WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
May be worth pulling this in as its own PR; possibly using VCR or similar for easy recording and replay of cassettes.

Reply:
That would be a good idea.